src/scenes/index.js

```diff
@@ -10,21 +10,17 @@ import {
 import Home from './Home';
 import TradeHub from './TradeHub';
 
-class Root extends Component {
-  render() {
-    return (
-      <Router>
-        <div>
-          <Route exact path='/' render={(props) => (
-            <Home opts={this.props.opts} />
-          )}/>
-          <Route exact path='/tradehub' render={(props) => (
-            <TradeHub opts={this.props.opts} />
-          )}/>
-        </div>
-      </Router>
-    )
-  }
-}
-
-export default Root;
+export default class Root extends Component {
+  render = () => (
+    <Router>
+      <div>
+        <Route exact path='/' render={(props) => (
+          <Home opts={this.props.opts} />
+        )}/>
+        <Route exact path='/tradehub' render={(props) => (
+          <TradeHub opts={this.props.opts} />
+        )}/>
```

Comment:
I've run this through a code beautifier and it's preferred like this:
```javascript
<Route
  exact
  path="/tradehub"
  render={props => <TradeHub opts={this.props.opts} />}
/>
```
I kinda like it, it's more clean